policies/bundles/AnomalyDetectionOutput/metric.rego

```diff
@@ -6,12 +6,12 @@ default applicable = false
 
 default compliant = false
 
-ad := input.anomalyDetection
+output := input.anomalyDetection.output
```

Comment:
```suggestion
output := input.anomalyDetection.applicationLogging.logging.loggingService
```

Reply:
I changed it (without ".logging." as discussed) but I'm not 100% sure if we set this right in the ontology. Anyway, we will figure it out when we will try to implement it.